src/components/IconSwitch/_icon-switch.scss

```diff
@@ -0,0 +1,52 @@
+@import '../../globals/vars';
+
+$icon-switch-size-small: 32px;
+$icon-switch-size-default: 40px;
+$icon-switch-size-large: 48px;
```

Comment:
What if we used the Carbon spacing variables? They're already predefined and have the same values. I'm pretty sure the Icon sets themselves in Carbon use them under the hood for sizing styles.

`$icon-switch-size-small: 32px;` ->  `$spacing-07`
`$icon-switch-size-default: 40px;` -> `$spacing-08`
`$icon-switch-size-large: 48px;` -> `$spacing-09`

Reply:
Nice :D done